elasticdl_client/common/args.py

```diff
@@ -189,6 +189,12 @@ def add_train_params(parser):
         help="The command executed in the pod launched by the master",
         default="",
     )
+    parser.add_argument(
```

Comment:
Will we public this arguments to the user?

Reply:
No need. Deleted.